src/app.js

```diff
@@ -1 +1,34 @@
+/* eslint-disable no-console */
 'use strict';
+
+const args = process.argv.slice(2);
+
+const operation = args[0];
+
+const numbers = args.slice(1).filter(arg => !isNaN(arg)).map(Number);
+
+if (!numbers.length) {
+  switch (operation) {
+    case 'add' || 'multiply':
+      console.log(`There is nothing to ${operation}`);
+      break;
```

Comment:
```suggestion
    case 'add'
    case 'multiply':
      console.log(`There is nothing to ${operation}`);
      break;
```
It is better to combine cases like this